src/main/java/com/kodilla/ecommercee/domain/Group.java

```diff
@@ -0,0 +1,15 @@
+package com.kodilla.ecommercee.domain;
+
+import org.jetbrains.annotations.NotNull;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "PRODUCT_GROUPS")
+public class Group {
+
+    @Id
+    @NotNull
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
Still Identity.

Reply:
Class Group doesn't belong to me, so technicly there will be changes.